### ISSUE TITLE
Update Roadmap for 2024

### ISSUE THREAD

**Issue by maddes8cht** [microsoft/terminal#17008]
The Roadmap needs to be updated again.  #15661 updated the roadmap for 2023 near the end of the year.
It' s 2024 now.
Isn't a radmap supposed to show the road ahead, not only backwards?

**Comment by maddes8cht** [comment]
The roadmap still tells me about "up next in the terminal" what i migth get in an upcoming version 1.19, which was ouf of preview thre months ago.

There's something embarrassing about that.

**Comment by maddes8cht** [comment]
So arround once a year someone opened an issue concerning the outdated Roadmap which gets closed immediately as duplicates.

Now another year is gone.
Obviously, no one is responsible for maintaining the roadmap, and no one is willing to update it.
I believe that as long as there is no solution to this, it would be better to remove the roadmap completely than to continue referring to this version, which has been outdated for years.

**Comment by Frulfump** [comment]
Yes and the fact that there's been zero comments to this issue from the maintainers even if they've changed tags and milestones is embarrassing. Sadly I encounter this in essentially all Microsoft repos where either issue triage or PR review is left behind.

**Comment by maddes8cht** [comment]
In fact, this “Roadmap 2023” is embarrassing in so many ways that it's astonishing how little a company like Microsoft cares about its public image.
It is linked on the front “readme” with the statement “The plan for the Windows Terminal is described here and will be updated as the project proceeds.” And then you end up on a page that is almost three years old, explaining the plans for 2023.
Not only has every “future release” mentioned there long since reached its EOL.
It also says:
“We've settled on a roughly quarterly release cycle – about once every three months.”
GA version 1.19 was actually released in 2023.
In 2024, releases 1.20 and 1.21 were released, and in 2025, 1.22 and 1.23 were released. We are now waiting for GA 1.24 in 2026. The GA release of 1.22 in February 2025 was 13 months ago. It is not a “quarterly release cycle,” but (not quite) two releases per year.

In principle, that's all fine. But then you should at least make sure that you don't prominently display an outdated “roadmap” with such statements, but rather (silently...) remove it after a few years to be on the safe side if no one has time to update it.

Sometimes “no roadmap” is better than one that is so hopelessly outdated. That doesn’t paint a good picture. Really.

**Comment by DHowett** [comment]
Alright, fine. You want action on this issue? Here it is.

**Comment by DHowett** [comment]
We've undergone more than a few changes in PM leadership over the past couple of years, as well as a change in PM job role to be less hands-on in this specific way. That leaves the engineering team (already spread pretty thin) to pick up the slack for that too.

I'm the person who chose to skip the mid-quarterly releases in '24 and '25, because we were trying to hit some Windows check-in deadlines for Stable and I didn't think what we had in Preview was up to the quality standard for promotion _to_ Stable which I wanted us to deliver into Windows. I probably could have communicated that better with the community, yes.

**Comment by DHowett** [comment]
In the meantime, and until we have better support, I'm removing the roadmap link from the README. There's probably still some durable things in there, which we'll have to evaluate. Eventually.

**Comment by DHowett** [comment]
(Also, on reflection: I'm sorry I was crabby about it!)
